gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/event/ApiEvent.java

```diff
@@ -23,4 +23,8 @@ public enum ApiEvent {
     DEPLOY,
     UPDATE,
     UNDEPLOY,
+    DYNAMIC_PROPERTY_STARTED_V2,
+    DYNAMIC_PROPERTY_STOPPED_V2,
+    DYNAMIC_PROPERTY_STARTED_V4,
+    DYNAMIC_PROPERTY_STOPPED_V4,
```

Comment:
```suggestion
    START_DYNAMIC_PROPERTY_V2,
    STOP_DYNAMIC_PROPERTY_V2,
    START_DYNAMIC_PROPERTY_V4,
    STOP_DYNAMIC_PROPERTY_V4,
```

Event are infinitive verb we should keep the consistency, even if I agree that an event should describe something that has happen 🙈